lib/pacemaker/pcmk_output.c

```diff
@@ -1554,14 +1554,15 @@ inject_rsc_action_xml(pcmk__output_t *out, va_list args)
     }
 
 PCMK__OUTPUT_ARGS("cluster-status", "pe_working_set_t *", "crm_exit_t", "stonith_history_t *",
-                  "gboolean", "uint32_t", "uint32_t", "const char *", "GList *", "GList *")
+                  "enum pcmk__fence_history", "uint32_t", "uint32_t", "const char *", "GList *",
+                  "GList *")
 int
 pcmk__cluster_status_text(pcmk__output_t *out, va_list args)
 {
     pe_working_set_t *data_set = va_arg(args, pe_working_set_t *);
     crm_exit_t history_rc = va_arg(args, crm_exit_t);
     stonith_history_t *stonith_history = va_arg(args, stonith_history_t *);
-    gboolean fence_history = va_arg(args, gboolean);
+    enum pcmk__fence_history fence_history = va_arg(args, enum pcmk__fence_history);
```

Comment:
A quirk of C is that for variadic functions, char, bool, and enum are promoted to int. So we should use va_arg(args, int) here (and anywhere else we're currently specifying an enum) like you do with the bools later.

It would be nice if gcc or static analysis could warn about such things ...

Reply:
With how I've got vim set up, I was seeing warnings about bool.  However, it doesn't warn about the enum.  These are the only uses of it.  Fixed.

Reply:
What are you using in vim for that?